### PR TITLE
Order display cards + count on dashboard

### PR DESCRIPTION
**Pull request by thanhlongnguyenn**

### CODE REVIEW

**Review by DevelopmentHF — CHANGES_REQUESTED** [review]
Num orders on main dashboard page is still showing as 2. It should be 3.

image

File: frontend/medpantry/components/Table 3.tsx
Comment (DevelopmentHF) [inline frontend/medpantry/components/Table 3.tsx:1]: Why is this duplicated in the other files

**Review by DevelopmentHF — CHANGES_REQUESTED** [review]
Duplicated code Table 2 Table 3 ?

**Comment by stakayanagi407** [comment]
Remade the Order into component. Also the Orders I think present the correct information now.

**Review by DevelopmentHF — APPROVED** [review]
Lgtm. Solid refactoring

**Pull request merged**